files/orchestrate/.env

```diff
@@ -1,18 +1,16 @@
 ## Orchestrate services endpoint hosts
-CHAIN_REGISTRY_HOST=http://localhost:8011
-CONTRACT_REGISTRY_HOST=localhost:8020
-TX_SCHEDULER_HOST=http://localhost:8031
+API_HOST=http://localhost:8031
 KAFKA_HOST=localhost:9092
 
 # Once an account created on the network, you can use it as a faucet account.
 # insert this account address here before running the `npm run create-faucet`
-FAUCET_ACCOUNT=0xf8369cf1948CD1bF438f45720C9f81ecB2909f7F
+FAUCET_ACCOUNT=[FAUCET_ACCOUNT]
```

Comment:
Is there a check somewhere that ensures that the user has reviewed this file and set up the correct address, or will they just see a confusing error?

Reply:
Same Q for the other edits in this file.

Reply:
The error won't be confusing but will be something of the sort: `Invalid From account` or `Invalid creditor address`. Given that the `.env` must be filled and that is explained in the README, I see no problem with this. Also, it would have failed as well before the changes since these hardcoded accounts don't exist and are invalid anyways.